src/py42/_internal/archive_access.py

```diff
@@ -159,15 +164,21 @@ def is_job_complete(self, job_id):
         response = self._storage_archive_client.get_restore_status(job_id)
         return self._get_completion_status(response)
 
-    def _start_restore(self, file_selection):
+    def _start_restore(self, file_selection, exceptions):
+        num_files = sum([fs.num_files for fs in file_selection])
+        num_dirs = sum([fs.num_dirs for fs in file_selection])
+        size = sum([fs.size for fs in file_selection])
+        zip_result = _check_for_multiple_files(file_selection) or None
         return self._storage_archive_client.start_restore(
-            self._device_guid,
-            self._archive_session_id,
-            file_selection.path_set,
-            file_selection.num_files,
-            file_selection.num_dirs,
-            file_selection.size,
+            guid=self._device_guid,
+            web_restore_session_id=self._archive_session_id,
+            path_set=[fs.path_set for fs in file_selection],
+            num_files=num_files,
+            num_dirs=num_dirs,
+            size=size,
+            zip_result=zip_result,
             show_deleted=True,
+            exceptions=exceptions,
```

Comment:
I would call this something like `exclusions` to avoid it being associated with the programming concept of an exception (error).